src/ResourceManager/Insights/Commands.Insights/help/Add-AzureRmLogProfile.md

```diff
@@ -46,7 +53,11 @@ Accept wildcard characters: False
 ```
 
 ### -Locations
-Specifies the list of locations.
+Specifies the location of the log profile.
+Valid values: Run below cmdlet to get the latest list of locations. 
+```yaml
```

Comment:
@angshumannayak please remove this extra ` ```yaml ` block. It will affect how `platyPS` generates the MAML from this markdown. You can keep the command you have in the block, but the ` ```yaml ` text string (and closing ` ``` `) will need to be removed.